src/ui/CategoryFacet/CategoryFacet.ts

```diff
@@ -400,38 +304,16 @@ export class CategoryFacet extends Component implements ICategoryFacet {
     }
   }
 
-  private tryToInitFacetSearch() {
-    // TODO: add new facet search
-    return this.logDisabledFacetSearchWarning();
-  }
-
-  private logDisabledFacetSearchWarning() {
-    if (!this.options.enableFacetSearch) {
-      return;
-    }
-
-    const valueCaptionAttributeName = this.getOptionAttributeName('valueCaption');
-    const enableFacetSearchAttributeName = this.getOptionAttributeName('enableFacetSearch');
-    const field = this.options.field;
-
-    this.logger.warn(`The search box is disabled on the ${field} CategoryFacet. To hide this warning,
-    either remove the ${valueCaptionAttributeName} option or set the ${enableFacetSearchAttributeName} option to "false".`);
-  }
-
-  private getOptionAttributeName(optionName: keyof ICategoryFacetOptions) {
-    return ComponentOptions.attrNameFromName(optionName);
-  }
-
   private get isCategoryEmpty() {
     return !this.values.allFacetValues.length;
   }
 
   private updateAppearance() {
     this.header.toggleCollapse(this.isCollapsed);
     $$(this.element).toggleClass('coveo-dynamic-category-facet-collapsed', this.isCollapsed);
-    this.show();
+    $$(this.element).removeClass('coveo-hidden');
     this.dependsOnManager.updateVisibilityBasedOnDependsOn();
-    this.isCategoryEmpty && this.hide();
+    this.isCategoryEmpty && $$(this.element).addClass('coveo-hidden');
```

Comment:
Would it be cleaner to still have the `hide` and `show` methods as private methods? Alternatively, a toggle could be used like `setHidden` or a setter like `set hidden(shouldHide: boolean)`.

Reply:
We could but since it's only use in one place I refrained from keeping show/hide